fastapi/utils.py

```diff
@@ -68,6 +77,29 @@ def create_response_field(
 ) -> ModelField:
     """
     Create a new response field. Raises if type_ is invalid.
+
+    Args:
+        name (str): The name of the response field.
```

Comment:
Why you use different style of describing parameters than in `get_path_param_names` method?